src/Microsoft.Health.Fhir.CosmosDb.UnitTests/Features/Operations/Reindex/ReindexJobCosmosThrottleControllerTests.cs

```diff
@@ -46,7 +46,7 @@ public ReindexJobCosmosThrottleControllerTests(ITestOutputHelper output)
             _fhirRequestContextAccessor.RequestContext = fhirRequestContext;
         }
 
-        [Fact]
+        [Fact(Skip = "TODO: Investigate intermittent failures")]
```

Comment:
Is there a work item for this so it doesn't get lost?

Reply:
Yes